tuned/plugins/plugin_modules.py

```diff
@@ -85,9 +87,14 @@ def _instance_verify_static(self, instance, ignore_missing, devices):
 			mpath = "/sys/module/%s" % module
 			if not os.path.exists(mpath):
 				ret = False
-				log.error(consts.STR_VERIFY_PROFILE_FAIL % "module '%s' is not loaded" % module)
+				emsg = consts.STR_VERIFY_PROFILE_FAIL % "module '%s' is not loaded" % module
+				log.error(emsg)
+				vlog.add_log(instance.name, module,
+                                                    emsg, result=ret)
 			else:
-				log.info(consts.STR_VERIFY_PROFILE_OK % "module '%s' is loaded" % module)
+				ret = True
```

Comment:
I still have problem with this. IE. instance._modules has keys = (module_a, module_b, module_c), module_a and module_b fails and module_c pass. In current implementation ret is False, because it's not overwrite to True again. In new implementation as module_c is processed last, ret is True and it changes result of this function.

Reply:
Fixed and repushed the commit.

Thank you.